src/lib/updating-element.ts

```diff
@@ -266,8 +271,8 @@ export abstract class UpdatingElement extends HTMLElement {
   /** @nocollapse */
   private static _ensureClassProperties() {
     // ensure private storage for property declarations.
-    if (!this.hasOwnProperty(
-            JSCompiler_renameProperty('_classProperties', this))) {
+    if (!Object.prototype.hasOwnProperty.call(
```

Comment:
I'm not a huge fan of this rule. We don't expect someone else to be monkey-patching this class or overriding `hasOwnProperty`, and if they do 🤷‍♂️.